### PR TITLE
Cleanup and local testing

### PR DESCRIPTION
**Pull request by ineiti**
Cleanup:
- renamed webserver to webapp
- removed conodes.toml from git, as this should be in the local installation only

Local testing:
- added Defaults.AssetsPath, so that conodes.toml can be found in local testing

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "~8.9.4",
     "codelyzer": "~4.5.0",
+    "cross-fetch": "^3.0.4",
```
Comment (tharvik) [inline webapp/package.json]: It is only used in dynacred, not in webserver, so it shouldn't be in devDeps

Comment (ineiti) [reply webapp/package.json]: but if I do `make dynacred_src`, then I need it. But only in dev, not in production, as the latter doesn't (shouldn't) depend on `make dynacred_src`...

**Comment by ineiti** [comment]
> removing conodes.toml means that we have to provide it on c4dtsrv before merging

Absolutely correct.

**Comment by ineiti** [comment]
I'm not sure, however, if a copied-in `conodes.toml` would survive `git clean -xdv` from `update.sh`. According to my reading of `man git-clean`, it would disappear.

File: webapp/package.json (hunk 1 above)
Comment (tharvik) [reply webapp/package.json]: Ha right, good for me then

**Comment by tharvik** [comment]
> I'm not sure, however, if a copied-in `conodes.toml` would survive `git clean -xdf` from `update.sh`. According to my reading of `man git-clean`, it would disappear.

It would disappear; we can copy it after `git clean`, as we are already doing some srcs modifications

**Comment by ineiti** [comment]
> removing conodes.toml means that we have to provide it on c4dtsrv before merging

Or we define the checked out directory as _clean_ and remove the `git clean` command ;) Both are fine for me.

**Comment by tharvik** [comment]
> > removing conodes.toml means that we have to provide it on c4dtsrv before merging
> 
> Or we define the checked out directory as _clean_ and remove the `git clean` command ;) Both are fine for me.

Hum, I was thinking of moving `update.sh` inside the repo at some point, so that travis does nearly all the job, and as we don't want to bundle a `conodes.toml`, we would still need a deploy script on the server copying the wanted toml. So I'm leaning toward the first fix (copying it afterward).

**Comment by ineiti** [comment]
OK, probably we still need to work on a more _real_ deployment scenario with the angular projects, where we can actually `npm i` it or something... using `git pull` and then `ng build` seems strange. What's the 'normal' angular way?

**Comment by tharvik** [comment]
> OK, probably we still need to work on a more _real_ deployment scenario with the angular projects, where we can actually `npm i` it or something... using `git pull` and then `ng build` seems strange. What's the 'normal' angular way?

From discussion on matrix, it seems that using `ng build --prod` then copying the `dist` dir to server is the way to go.

**Comment by tharvik** [comment]
`cleanup_local_testing-fw` contains a forwarded version.

**Comment by ineiti** [comment]
Close in favour of #43

**Pull request closed without merging**